src/utils.spec.ts

```diff
@@ -0,0 +1,54 @@
+import * as assert from 'assert';
+import { getOptions, isNullOrEmpty } from './utils';
+import { Options } from './validate';
+
+describe('utils', () => {
+
+    describe('isNullOrEmpty', () => {
+
+        it('should return true when empty string is passed', () => {
```

Comment:
Just a cosmetic suggestion, but if I'm not mistaken, in most tests in the CLI we use _it returns..._ instead of _it should return..._

Reply:
Happy to make the change for the sake of consistency.